Classes/Service/NodePolicyService.php

```diff
@@ -249,6 +249,8 @@ public function getDisallowedProperties(NodeInterface $node): array
         };
 
         $disallowedProperties = array_filter(array_keys($node->getNodeType()->getProperties()), $filter);
+        $disallowedProperties = array_values($disallowedProperties);
```

Comment:
hi :D why has this been added?

Reply:
`array_filter` leaves keys intact. Through `json_encode` this can lead to arrays being serialized as objects. `array_values` makes sure that we always end up with a strictly numerical array.